server/protocols/webadmin-integration-test/distributed-webadmin-integration-test/src/test/java/org/apache/james/webadmin/integration/rabbitmq/RabbitMQWebAdminServerTaskSerializationIntegrationImmutableTest.java

```diff
@@ -119,6 +119,27 @@ void fullReindexingShouldCompleteWhenNoMail() {
             .body("additionalInformation.messageFailures", is(anEmptyMap()));
     }
 
+    @Test
+    void fixPop3InconsistenciesShouldCompleteWhenNoMail() {
+        String taskId = with()
+            .post("/mailboxes?task=fixPop3Inconsistencies").prettyPeek()
```

Comment:
These two tests are on top of a full fledged James server, everything is fine, bound and exposed ;-)